src/java.base/share/classes/java/lang/Module.java

```diff
@@ -109,8 +110,10 @@
     // the module descriptor
     private final ModuleDescriptor descriptor;
 
-    // true, if this module allows restricted native access
-    private volatile boolean enableNativeAccess;
+    // true, if this module allows restricted native access; @Stable makes sure that modules that allow native
+    // access capture this property as a constant.
```

Comment:
Do you mind fixing this comment to avoid the really long line, it sticks out compare to everything else around it.